pkg/cmd/server/api/types.go

```diff
@@ -1202,3 +1202,13 @@ type ServiceServingCert struct {
 	// If this value is nil, then certs are not signed automatically.
 	Signer *CertInfo
 }
+
+// AdmissionPluginActivation can be used to enable to disable various admission plugins.
+// When this type is present as the `configuration` object under `pluginConfig` and *if* the admission plugin supports it,
+// this will cause an off by default admission plugin to be enabled
+type AdmissionPluginActivation struct {
```

Comment:
Ok, that's slightly better then.

On May 23, 2016, at 8:38 PM, David Eads <notifications@github.com> wrote:

In pkg/cmd/server/api/types.go
<https://github.com/openshift/origin/pull/8945#discussion_r64312661>:

> @@ -1202,3 +1202,13 @@ type ServiceServingCert struct {
>  	// If this value is nil, then certs are not signed automatically.
>  	Signer *CertInfo
>  }
> +
> +// AdmissionPluginActivation can be used to enable to disable various admission plugins.
> +// When this type is present as the `configuration` object under `pluginConfig` and *if* the admission plugin supports it,
> +// this will cause an off by default admission plugin to be enabled
> +type AdmissionPluginActivation struct {

I don't like that it's called pluginConfig (for starters).

It's inside of a stanza called admissionConfig, so if we changed the name
it would be admissionConfig.admissionConfig which is probably worse than
the current admissionConfig.pluginConfig.

—
You are receiving this because you are on a team that was mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/8945/files/e3efce5d11abdec554a2cda21f8b72dd093473e6#r64312661>